File: dotnet/cafdemalihapa/Directories/Repository/DotnetDirectory/DotnetDirectory.cs
namespace cafdemalihapa.Directories.Repository.DotnetDirectory
{
    public class DotnetDirectory
    (
        RepositoryDirectory repositoryDirectory
    )
    {
        public string GetPath()
        {
            var repositoryDirectoryPath = repositoryDirectory.GetPath();
            return Path.Combine(repositoryDirectoryPath, "dotnet");
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Repository/Dotnet/DotnetDirectory.fs
namespace cafdemwimu.console.Directories.Repository.Dotnet

open System.IO
open cafdemwimu.console.Directories.Repository

type DotnetDirectory(repositoryDirectory: RepositoryDirectory) =
    member _.GetPath() =
        let repositoryDirectoryPath = repositoryDirectory.GetPath()
        Path.Combine(repositoryDirectoryPath, "dotnet")
